src/app/(main)/layout.tsx

```diff
@@ -26,7 +26,7 @@ export default async function Layout({ children }: LayoutProps) {
       <div className="relative flex flex-1 overflow-hidden">
         <Sidebar />
         <main className="flex w-full flex-1 flex-col items-center overflow-y-auto p-10">
-          <div className="flex w-full max-w-[1500px] flex-col gap-y-10">
+          <div className="flex h-full w-full max-w-[1500px] flex-col gap-y-10">
```

Comment:
I think this broken the styling on all of the voyage page https://i.imgur.com/MMZKIW6.png can you fix this? It's hard to see but the issue is there's no margin on the bottom of the cards anymore. This is in all of the relevant voyage pages btw

Reply:
@Dan-Y-Ko removed all these `h-full` from layouts, don't know what was wrong with the bottom padding... so I decided to center the error component with some math...